loomi/regulariza-diagonal-client
# Reports page

## Reports page
PS: Only review after feature/modals be reviewed
---

**What I did:**

- Reports page
  - FeatureReport
  - SituationReport
  - Socioeconomic report
- Charts: BarChart, LineChart, DoughnutChart

Missing:
- Mock data
- BarChart tooltip

**How to test:**

1. In your browser go to [stage](http://project.netlify.app/) or [localhost](http://localhost:3000/)
2. Check if the component/view looks just like the [mockup](https://xd.adobe.com/) in different screen sizes (width between 1024px and 1920px)
3. Test the things that you think are worth testing, even the ones that are not in this description

![PRReport1](https://user-images.githubusercontent.com/45184938/125143648-a41c7280-e0f1-11eb-9094-7bbc68e6e4cb.png)
![PRReport2](https://user-images.githubusercontent.com/45184938/125143653-a8e12680-e0f1-11eb-8ee1-6bb6d327e7b8.png)
![PrReport3](https://user-images.githubusercontent.com/45184938/125143657-aa125380-e0f1-11eb-9541-549bffdc15b2.png)
